src/main/resources/templates/index.html

```diff
@@ -287,7 +287,7 @@ <h4 class="card-title" th:text="#{page.deploy}"></h4>
             	setTimeout(function () {
                 	hidePleaseWait();
                 	goToEndStep();
-            	}, 3000)
+            	}, 5000)
```

Comment:
why increasing this?

Reply:
as soon as this timeout is over we show the "Success" part of the page and user thinks like all is ok ..just after a second to see the error page. was not looking good to me. The 5 seconds covers the initial error check and the first 4 second check for process errors where i think majority of issues will be. wdyt? I will look into updating this part of the js soon so its more stable than having a simple timeout but for now this is best i could think of.